include/knowhere/config.h

```diff
@@ -382,13 +382,14 @@ class Config {
     EntryAccess<decltype(PARAM)> PARAM##_access(std::get_if<Entry<decltype(PARAM)>>(&__DICT__[#PARAM])); \
     PARAM##_access
 
+const float default_range_filter = 1.0f / 0.0;
```

Comment:
nit: Let's follow https://zilliverse.feishu.cn/docs/doccnYCofnA36k6zRy37TX4tMYo use `kCamalCase` for constants.

Reply:
updated